Server/Entities/LoginRequest.go

```diff
@@ -0,0 +1,5 @@
+package entities
+
+type LoginRequest struct {
+	UserName string `json:"userName" bson:"userName"`
```

Comment:
Change bson and saving on client to user_name

Reply:
so the tag should be `json:"userName" bson:"user_name"`?

Reply:
yea, and save it in the server as user_name

Reply:
when I'll get to saving it to the server :)

Reply:
fixed